### ISSUE TITLE
Application?

### ISSUE THREAD

**Issue by timqian** [johannesmik/neurons#17]
Is it possible to accomplish some application such as pattern recognition using this framework?

**Comment by johannesmik** [comment]
Yes I think it would be possible, however we haven't used it for this purpose yet. What kind of pattern recognition do you want to do and in which scale? Maybe I can help you.

Also note, that this code simulates 'real' neurons using the [SRM](http://www.scholarpedia.org/article/Spike-response_model) and the [STDP](http://www.scholarpedia.org/article/STDP) model. Simulation is rather slow. Maybe for your application artifical neurons might be better suited.

**Comment by timqian** [comment]
such as [Distracted Sequence Recall](http://synaptic.juancazala.com/#/dsr). this example is accomplished by LSTM, but I think it can be done by SNN too.
And [this article](http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=6760966&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D6760966) using snn and stdp training an insect. I am learning snn reasently, and find your explain nation very clear. Thanks for your work.
